### PR TITLE
Draft datastorage API

### PR DESCRIPTION
**Pull request by freliciatucker**
- please Disregard  FileUploard Servlet and index.html
- attempts to create API for David to use to encode and decode blobs

### CODE REVIEW

File: src/main/java/com/google/cloud/storage/Database.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.google.cloud.storage;
+
+import java.util.HashMap;
+
+
+/* This class creates an instance of a database that a file can be written to and read from.
```
Comment (zjwoody) [inline src/main/java/com/google/cloud/storage/Database.java:6]: Use block comments:
https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Comment (zjwoody) [inline src/main/java/com/google/cloud/storage/Database.java:6]: It is not creates an instance, it is the database api

File: src/main/java/com/google/cloud/storage/Database.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Assumptions: for weach runID, only 1 of each datatype can be associated with it */
+public class Database{
```
Comment (zjwoody) [inline src/main/java/com/google/cloud/storage/Database.java:8]: I think you may need to update the code by our yesterday's discussion.

Comment (freliciatucker) [reply src/main/java/com/google/cloud/storage/Database.java:8]: will be meeting today with David to discuss changes!

**Pull request closed without merging**